.gitignore

```diff
@@ -1,12 +1,8 @@
-.idea
-.DS_Store
-build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
-composer.lock
-vendor
-node_modules
-.sass-cache
+/.sass-cache
+/build
+/Resources/doc/_build
+/node_modules
```

Comment:
Though that these folders are project specific. There is SCSS files and a package.json un the Resources folder...